file-formats/tobj/README.md

```diff
@@ -0,0 +1,6 @@
+# TOBJ File Format
+This AFF is only complete with regard to ABAP Language Version 'ABAP for Cloud Development'
+
+File | Cardinality | Definition | Schema | Example
+:--- | :--- | :--- | :--- | :---
+`<name>.tobj.json` | 1 | [`zif_aff_tobj_v1.intf.abap`](./type/zif_aff_tobj_v1.intf.abap) | [`tobj-v1.json`](./tobj-v1.json) | [z_aff_example_tobjt.tobj.json](./examples/z_aff_example_tobjt.tobj.json)
```

Comment:
Not sure whether `z_aff_example_tobjt` shall be `z_aff_example_tobj` 🤓

Reply:
Hi Albert,
the name of a TOBJ must end with its type e.g. T or L
when the TOBJ is created you must specify its type and you cannot change it afterwards
e.g.
![image](https://github.com/SAP/abap-file-formats/assets/105780276/c36bf200-fd53-49c6-8129-6eb61d50d7de)